key-rotator/main.go

```diff
@@ -370,11 +370,12 @@ func writeManifests(
 		ingestor, oldManifest, newManifest := ingestor, oldManifest, newManifestByIngestor[ingestor]
 		eg.Go(func() error {
 			if oldManifest.Equal(newManifest) {
+				log.Debug().Msgf("Skipping write for manifest for (%q, %q): key unchanged", locality, ingestor)
```

Comment:
As in #1152, could you put locality and ingestor in separate string fields on the structured message for ease of searching?

Reply:
Done -- as in that PR, I also keep `locality`/`ingestor` in the unstructured log message, because at least some interfaces do not surface the structured portion of the log.